RELEASE.md

```diff
@@ -6,6 +6,12 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
+# Release 6.6.2
+
+## Major features and improvements
+
+- Reduce aggressiveness of dependabot's versioning strategy (#1602)
```

Comment:
We don't need this in the release notes. This information should only be user focused, and this wouldn't fall into that realm.

Reply:
Ah that makes sense! Good call, will remove then yeah